<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Maditor::View::ConfigWidget</class>
 <widget class="QWidget" name="Maditor::View::ConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="executableBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Executable</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="maditorLauncherButton">
        <property name="text">
         <string>Maditor-Launcher</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">launcherGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="customLauncherButton">
        <property name="text">
         <string>Custom:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">launcherGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="customLauncherCmd">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="launcherSettings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Maditor-Launcher-Settings</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="clientButton">
        <property name="text">
         <string>Client</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">launcherTypeGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="serverButton">
        <property name="text">
         <string>Server:</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">launcherTypeGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="serverName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="modulesLabel">
        <property name="text">
         <string>Modules:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QListView" name="modulesView"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>customLauncherButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>customLauncherCmd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>maditorLauncherButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>launcherSettings</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>72</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>serverButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>serverName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="launcherGroup"/>
  <buttongroup name="launcherTypeGroup"/>
 </buttongroups>
</ui>
